### ISSUE TITLE
only operate on components in /components folder

### ISSUE THREAD

**Issue by faceyspacey** [faceyspacey/remixx#9]
recursively checking of course. 

with configuration options for both `include` + `exclude`. both which are arrays of paths.

**Comment by faceyspacey** [comment]
...if we're gonna be using this babel plugin any time soon, we gotta make sure it doesn't affect functions it shouldnt affect. ...i know u got that return JSX stuff. i guess that might handle the real goal here. But bottom line: we need tight control over what we transform early on. 

Truthfully brother, this whole thing is super close to being ready for the most challenging part of merging routes into one big `ROUTES_MANIFEST`. Which means we'll be using this soon.

Deadline for all the to dos is a weak. ..I may not write the article on Rudy. I'll just release it to Daniel and let them all get the value they seem to want now from it. It will be good testing. But i think the right play isnt to stick our head out with promotional articles until Respond Framework is done. It's better not to confuse people with complex messaging. 

In short, Im jumpin on this with u asap.
